src/utils/contracts_consts.js

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import ApolloDepositStoreJson from '../../contracts/ApolloDepositStore.json';
+import AtlasStakeStoreJson from '../../contracts/AtlasStakeStore.json';
+import BundleStoreJson from '../../contracts/BundleStore.json';
+import ChallengesJson from '../../contracts/Challenges.json';
+import ConfigJson from '../../contracts/Config.json';
+import FeesJson from '../../contracts/Fees.json';
+import KycWhitelistJson from '../../contracts/KycWhitelist.json';
+import PayoutsJson from '../../contracts/Payouts.json';
+import PayoutsStoreJson from '../../contracts/PayoutsStore.json';
+import RolesJson from '../../contracts/Roles.json';
+import RolesStoreJson from '../../contracts/RolesStore.json';
+import ShelteringJson from '../../contracts/Sheltering.json';
+import ShelteringTransfersJson from '../../contracts/ShelteringTransfers.json';
+import TimeJson from '../../contracts/Time.json';
+import UploadsJson from '../../contracts/Uploads.json';
+
+const contractJsons = {
+  time: TimeJson,
+  atlasStakeStore: AtlasStakeStoreJson,
+  roles: RolesJson,
+  bundleStore: BundleStoreJson,
+  kycWhitelist: KycWhitelistJson,
+  sheltering: ShelteringJson,
+  fees: FeesJson,
+  challenges: ChallengesJson,
+  payoutsStore: PayoutsStoreJson,
+  payouts: PayoutsJson,
+  shelteringTransfers: ShelteringTransfersJson,
+  config: ConfigJson,
+  uploads: UploadsJson,
+  rolesStore: RolesStoreJson,
+  apolloDepositStore: ApolloDepositStoreJson
```

Comment:
I would argue this should also include the Head, Context and all the other contracts. Alternatively a second list/map should be defined with those special contracts. 

This would centralise all imports for JSON files, and unify the naming.

Reply:
done 👍